common/flatpak-dir.c

```diff
@@ -8189,7 +8247,7 @@ flatpak_dir_uninstall (FlatpakDir                 *self,
   parts = flatpak_decompose_ref (ref, error);
   if (parts == NULL)
     return FALSE;
-  name = parts[1];
+  app = parts[1];
```

Comment:
Why did this change to app? It could be a runtime too, no?

Reply:
I guess I wanted to keep 'name' in sync with the other changes in this commit. But I can change it back, no problem